lib/src/commands.rs

```diff
@@ -1357,17 +1361,26 @@ impl Command for QuitCommand {
                 .expect("error unwraping stdout of ps");
             let child_processes = owned_child_processes.split("\n").collect::<Vec<&str>>();
 
-            let mut spawned_pids: Vec<u32> = Vec::new();
+            // &str representation of PIDs
+            let mut spawned_pids: Vec<&str> = Vec::new();
+
             for child in child_processes {
                 println!("child: {}", &child);
                 if !child.is_empty() {
                     let ch: Vec<&str> = child.trim_start().split_whitespace().collect();
-                    println!("ch: {}", &ch[0]);
-                    spawned_pids
-                        .push(u32::from_str_radix(ch[0], 10).expect("error with from_str_radix"));
+                    println!("ch[0]: {}", &ch[0]);
+                    spawned_pids.push(ch[0]);
                 }
             }
+
+            for pid in spawned_pids {
+                Command::new("kill")
```

Comment:
I am not sure, but I didn't want to deal with the scoping issues of this (for now) - the process is made in the regtest module, and spun off really with no handles. I definitely thought this would be the elegant way, but I didn't want to fuss with it, I suggest a follow-up issue to make things nicer?